resources/editors/codemirror.html

```diff
@@ -6,6 +6,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <link rel="stylesheet" href="/style.css" />
         <title>CodeMirror Test</title>
+        <script>Object.defineProperty(window, 'requestIdleCallback', {value: undefined, writeable: true});</script>
```

Comment:
Why did you do this differently than in https://github.com/WebKit/Speedometer/pull/242? Can we pick one way for all tests that need this removed?

Reply:
Oh, I wrote this one first before seeing code for overriding requestAnimationFrame in next/nuxt. We can match that here.